tests/stub/routing.py

```diff
@@ -2011,12 +2276,12 @@ def test_should_ignore_system_bookmark_when_getting_rt_for_multi_db(self):
 
     def test_should_request_rt_from_all_initial_routers_until_successful(self):
         # TODO add support and remove this block
-        if get_driver_name() in ['dotnet', 'javascript']:
+        if get_driver_name() in ['dotnet']:
             self.skipTest("needs ROUTE bookmark list support")
-        if get_driver_name() in ['python', 'javascript', 'go', 'dotnet']:
+        if get_driver_name() in ['go', 'dotnet']:
```

Comment:
Good to see that python and javascript implement DNS discovery properly 👍

Reply:
:tada: